framework/packages/abstract-sdk/src/lib.rs

```diff
@@ -7,7 +7,7 @@
 pub type AbstractSdkResult<T> = Result<T, crate::error::AbstractSdkError>;
 
 /// The Abstract Core crate which contains the state and message objects for the native contracts. Also contains helper objects.
-pub extern crate abstract_core as core;
+pub use abstract_core as core;
```

Comment:
We should consider rename re-export it as `core` because of rust core library

Reply:
Yeah, that's why we don't re-export anymore...

Reply:
re-export could fix the "same type with different version" errors. For example users will only use `abstract-app` for the "App" and re-use `abstract-interface`, `abstract-sdk`, `abstract-core`, from it without dealing with weird version errors

Reply:
I'll create a separate task for this!

top level crates are abstract-app and abstract-adapter so you're saying they should both export abstract-sdk AND abstract-core?

Reply:
> AND cw-orchestrator

But this one could be probably re-exported by interface?